Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,150 @@
 /* 
 Add in your custom style such that you can develop the wonderful website within the exercise. 
 */
+
+html {
+    scroll-behavior: smooth;
+  }
+.firstOrangeDiv {
+    background-color:#f05f40;
+    padding-top:2%;
+    padding-bottom:1%;
+}
+
+.navbar-brand, .navbar, .navbar-light {
+    background-color: transparent;
+    transition: all, 0.35;
+}
+
+.navbar-brand.scrolled, .navbar.scrolled, .navbar-light.scrolled {
+    background: white !important;
+    color: orange;
+
+}
+
+a {
+    background: transparent;
+}
+
+.navbar-fixed-top {
+    background-color: #f0f0f0 !important;
+}
+
+.whiteText {
+    color: white;
+}
+
+.adjustedPicture {
+    max-width: 480px;
+}
+
+.jumbotron {
+    min-height:1150px;
+    padding-bottom: -15%;
+}
+
+#headerTitle {
+    font-size: 400%;
+    font-weight: bold;
+    text-align: center;
+    max-width: 50%;
+}
+
+#headerText {
+    margin-top: 15%;
+}
+
+p {
+    text-align: center;
+    max-width: 50%;
+}
+
+#orangeHR {
+    max-width: 50px !important;
+    border-color: #f05f40 !important;
+    border-width: 3px !important;
+}

```

Comment:
You can make a HR class, and then use id to just apply the color, this way you are repeating less code.